diff --git a/config/config.sub b/config/config.sub
index 61cb4bc22db8..f9bb3d51bf09 100755
--- a/config/config.sub
+++ b/config/config.sub
@@ -2,7 +2,7 @@
 # Configuration validation subroutine script.
 #   Copyright 1992-2013 Free Software Foundation, Inc.
 
-timestamp='2013-10-01'
+timestamp='2021-06-20'
 
 # This file is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -1508,6 +1508,10 @@ case $os in
 		;;
 	-nacl*)
 		;;
+
+        -serenity*)
+                os=-serenity
+                ;;
 	-none)
 		;;
 	*)
